### PR TITLE
Plant trial export download

### LINKED ISSUE #683: Compress Zenodo export to three files
Join `trait_scoring` with `plant_scoring_units` and `plan_accessions`. Leave the other two files as they are.

### PR DESCRIPTION
**Pull request by Nuanda**
Fixes #683 #684 #690. Partially addresses #680 #639.

### CODE REVIEW

File: spec/services/submission/plant_trial_exporter_spec.rb [hunk 1]
```diff
@@ -21,40 +26,32 @@
...
+      generated_scores = CSV.parse(documents[:trait_scoring]).map{ |row| row[13,3] }[1,3]
+      expect(documents[:trait_scoring].lines[1,3].map{ |l| l.split(',')[0] }).
         to match_array psus.map(&:scoring_unit_name)
-      expect(documents[:trait_scoring].lines[1,2].map{ |l| l.split(',')[1] }).
-        to match_array [ts1.score_value, '-']
-      expect(documents[:trait_scoring].lines[1,2].map{ |l| l.split(',')[2] }).
-        to match_array [ts2.score_value, '-']
-      expect(documents[:trait_scoring].lines[1,2].map{ |l| l.strip.split(',')[3] }).
-        to match_array [ts3.score_value, '-']
+      expect(generated_scores.map{ |l| l[0] }).to match_array [ts1.score_value, '-', '-']
```
Comment (kammerer) [inline spec/services/submission/plant_trial_exporter_spec.rb]: I got the impression that order of scores should be important here, shouldn't it?

**Comment by Nuanda** [comment]
@kammerer  Both comments addressed. Please go ahead with the review.

**Comment by kammerer** [comment]
:+1:

**Comment by Nuanda** [comment]
@kammerer Tomasz, I have added one more commit - to make the CSV headers consistent with the Browse section naming, and also removed two useless columns in one of the documents. Please see the commit and check if I should change anything.

**Comment by Nuanda** [comment]
I'll go ahead and merge-deploy that since time is short and we want the client's feedback. Meanwhile I'll leave the branch so any further changes could be made on it.

**Comment by Nuanda** [comment]
Note to self:

- [ ] test escaping commas in textual fields
- [ ] do something with design factors field export

**Comment by Nuanda** [comment]
Moved all changes to #692

**Pull request merged**